style.css

```diff
@@ -1230,12 +2849,158 @@ body {
     letter-spacing: -.5px;
     margin-top: 384px;
   }
+
+  .second-page__article {
+    width: 270px;
+  }
+
+  .second-page__title {
+    font-size: 23px;
+  }
+
+  .second-page__subtitle {
+    font-size: 11px;
+    margin-bottom: 17px;
+  }
+
+  .second-page__description {
+    width: 265px;
+    margin-bottom: 20px;
+    font: 600 10px/14px 'Montserrat-Regular', sans-serif;
+  }
+
+  .second-page__color-options {
+    margin-bottom: 40px;
+  }
+
+  .eclipse {
+    width: 25px;
+    height: 25px;
+    margin-right: 13px;
+  }
+
+  .second-page__price {
+    margin-bottom: 38px;
+    font-size: 23px;
+  }
+
+  .second__add-to-cart {
+    font-size: 15px;
+    padding: 10px;
+  }
+
+  .third-page__wrap {
+    width: 54%;
+    left: 12%;
+    bottom: 33px;
+    margin-bottom: 40px;
+  }
+
+  .third-page__article {
+    padding: 25px 25px 28px;
+    width: 115%;
+  }
+
+  .third-page__title {
+    font-size: 24px;
+  }
+
+  .third-page__subtitle {
+    font-size: 11px;
+  }
+
+  .third-page__description {
+    width: 240px;
+    font-size: 10px;
+    line-height: 13px;
+  }
+
+  .third__add-to-cart {
+    font-size: 14px;
+    padding: 11px 16px;
+  }
+
+  .third__more-items {
+    margin-left: 72%;
+    width: 147px;
+    font-size: 16px;
+    top: 40px;
+  }
+
+  .third__more-vector {
+    height: 10px;
+  }
 }
 
 @media (max-width: 384px) {
   .wrap__heading {
     font-size: 50px;
   }
+
+  .second-page__article {
+    width: 251px;
+    padding: 13px 26px 22px 29px;
+    margin-bottom: 12%;
+  }
+
+  .second-page__title {
+    margin-bottom: 1px;
+    font-size: 20px;
+  }
+
+  .second-page__subtitle {
+    font-size: 9px;
+    margin-bottom: 15px;
+    letter-spacing: 1.1px;
+  }
+
+  .second-page__description {
+    width: 250px;
+    margin-bottom: 16px;
+    font: 600 9px/12px 'Montserrat-Regular', sans-serif;
+    letter-spacing: -.2px;
+  }
+
+  .second-page__color-options {
+    margin-bottom: 35px;
+  }
+
+  .eclipse {
+    width: 23px;
+    height: 23px;
+    margin-right: 12px;
+  }
+
+  .second-page__price {
+    margin-right: 7px;
+    margin-bottom: 20px;
+    font-size: 20px;
+  }
+
+  .second__add-to-cart {
+    font-size: 13px;
+    padding: 9px;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #EBE3D8 104px, #000 104px, #000 145px, #EBE3D8 145px);
```

Comment:
Codacy found an issue: [Expected "#EBE3D8" to be "#ebe3d8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)